docs/overview/terminology.md

```diff
@@ -1,34 +1,54 @@
-| **Provider**                           |
-| **Resource**                           |
-| **Resource Manifest**                  |
-| **Rule**                               | **Rules**<p>* bound to a **Contract Offer**, **Contract Agreement** or **Contract Offer Framework**<p>* exist independent from an **Asset**
-| **Transfer Process**                   | **Transfer Process**<p>* based on a **Contract Agreement**
+| Name                         | Description                                         |
+|:-----------------------------|:---                                                 |
+| **Artifact**                 | The instance of code which can be modulized used by the EDC
+| **Asset**                    | An **asset** describes **data** that can add value to a **dataspace** and contains the **dataAddress**
+| **Asset Index**              | **Asset Index**<p>* manages assets<p>* provided by an extension<p>* may support external catalogs<p>* can be queried 
+| **Broker**                   | see **IDS Broker**
+| **Catalog**                  | **Catalog** is a directory about all meta-information about the **assets** a **data consumer** can reach in the **dataspace**
+| **Configuration**            | The EDC allows using a configuration file which will be loaded at every startup of an EDC instance
+| **Connector**                | **Connector** is the central instance for a participant to join a **dataspace**. The **connector** is the gateway from the user to the **dataspace**.
+| **Contract**                 | **Contract** is the finalized meta-information for a data transfer after a **contract negotiation** including the **policies** who have to be followed
+| **Contract Managment**       | Includes all parts of **contract offer**, **contract agreement** and **contract negotiation**
+| **Contract Agreement**       | **Contract Agreement**<p>* points to a **Contract Offer**<p>* results from a **Contract Negotiation Process**<p>* has a start date and may have a expiry date and a cancellation date
+| **Contract Negotiation**     | * MVP: only possible to accept already offered contracts. Counter offers are rejected automatically.
+| **Contract Offer**           | **Contract offer**<p>* set of obligations and permissions<p>* generated on the fly on provider side (see **Contract Offer Framework**)<p>* are immutable<p>* persisted in **contract negotiation process** once the negotiation has started<p>
+| **Contract Offer Framework** | **Contract offer framework**<p>* generates **contract offer templates**<p>* provided by **extensions**<p>* may be implemented in custom **extensions** to created **contract offers** based on existing systems
+| **Contract Offer Template**  | Blueprint of a **contract offer**
+| **Consumer**                 | see **data consumer**
+| **Consumer Pull**            | The data transfer is initilazied by the Consumer
+| **Data**                     | **Data** describes the properties of business objects
```

Comment:
I have tried to reformulate it in the sense that data have the task to represent information. Honestly, however, a good description for data is to be found, because from different points of view data are always defined slightly differently. Hope it fits better now.